packages/x-charts/src/LineChart/AreaElement.tsx

```diff
@@ -87,6 +91,11 @@ AreaElementPath.propTypes = {
 
 export type AreaElementProps = Omit<AreaElementOwnerState, 'isFaded' | 'isHighlighted'> &
   React.ComponentPropsWithoutRef<'path'> & {
+    /**
+     * If `true`, animations are skiped.
```

Comment:
Nitpick: have you considered creating a `BaseChartsProps`(or something similar) type for this and similar props?
This suggestion also applies to all existing animated charts. 🙈 
```suggestion
     * If `true`, animations are skipped.
```